fladle-plugin/src/test/resources/android-project2/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
```

Comment:
is the contents of the file and this directory the same as what is in `android-project`? if so, we can likely just copy the files from there instead of pasting them again here.

Reply:
They're not quite the same, `android-project2/build.gradle` has a `fulladleModuleConfig` block. I didn't want to complicate the test by adding logic around copying `android-project` dir and modifying the `build.gradle` in the test set up.